packages/blocks/tests/php/test-blocks.php

```diff
@@ -0,0 +1,69 @@
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
+/**
+ * Test methods from Automattic\Jetpack\Blocks
+ *
+ * @since 9.0.0
+ *
+ * @package automattic/jetpack-blocks
+ */
+
+namespace Automattic\Jetpack;
+
+use Automattic\Jetpack\Blocks;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Class Test_Blocks
+ */
+class Test_Blocks extends TestCase {
+	/**
+	 * Test the different inputs and matching output for Classes.
+	 *
+	 * @since 9.0.0
+	 *
+	 * @covers Automattic\Jetpack\Blocks::classes
+	 */
+	public function test_block_classes() {
+		$block_name = 'foo';
+		$attr       = array(
+			'bar'       => 'baz',
+			'align'     => 'wide',
+			'className' => 'editorclass',
+		);
+		$extra      = array( 'extraclass' );
+
+		$block_classes = Blocks::classes( $block_name, $attr, $extra );
+
+		$this->assertContains( 'wp-block-jetpack-foo', $block_classes ); // a general class is created from the block name.
+		$this->assertNotContains( 'bar', $block_classes ); // The extra 'foo' attribute should be dropped.
+		$this->assertContains( 'alignwide', $block_classes ); // an alignment class is created.
+		$this->assertContains( 'editorclass', $block_classes ); // className classes are passed.
+		$this->assertContains( 'extraclass', $block_classes ); // Extra class remains.
+	}
```

Comment:
Should there also be a test for not having 'align' or 'className' or `$extra` still working correctly? And for an unrecognized value for 'align'?

Reply:
Sure thing. Done in d31702c6e4b837845be60e2dbd72e440d9a9f2d9 and 09d6816ce0dde9ca4d1f878548a9e1b64390d22f